codelab/01_kpt-validate/tutorial.md

```diff
@@ -0,0 +1,326 @@
+# Validate Your Configurations in skaffold Workflow
+
+<walkthrough-disable-features toc></walkthrough-disable-features>
+
+## Introduction
+
+#### What is kpt?
+
+Kpt is [an OSS tool](https://github.com/GoogleContainerTools/kpt) for Kubernetes packaging, which uses a standard format to bundle, publish, customize, update, and apply configuration manifests.
+
+#### What kpt can help you in skaffold
```

Comment:
Yeah, this is a duplicate content. I think it is fine because the README and tutorials are in different platform (GitHub, Cloud Shell) and it is necessary in both places to give users a basic idea what the codelab is about. Besides, we will refer the codelab in other places (e.g. kpt or other cloudshell codelab), so some users may read this content only from the cloud shell.

Reply:
Gotcha 👍🏼